share/spice/anime/content.handlebars

```diff
@@ -0,0 +1,24 @@
+<div class="detail__media  detail__media--anime">
```

Comment:
If you're using on of our built-in templates you shouldn't need to define a custom detail template. Using a normalize function in `Spice.add()` will allow you to set the properties the media template expects. It will automatically build everything for you.

We'd like devs to write as little handlebars as necessary. That way if we later update the design of the Media template group, it will automatically update for all the IAs using that template :)

> More Details: https://duck.co/duckduckhack/spice_templates_overview#productsitemdetail-template